pyproject.toml

```diff
@@ -140,8 +142,34 @@ flake8 = "3.8.4"
 black = "^22.6.0"
 # Latest version compatible with `trio_typing~=0.5`, required by qtrio
 mypy = "v0.910"
+
 setuptools = "^63.1.0"
+maturin = "~=0.13"
+cibuildwheel = "2.8.1"
+
+
+[tool.poetry.build]
+generate-setup-file = false
+script = "build.py"
+
+[tool.cibuildwheel]
+build = "cp39-{manylinux,macos,win}*"
+archs = ["auto64"]
+build-verbosity = 3
+before-all = "bash ci/setup-rust.sh"
+before-build = "python -m pip install -U pip"
+
+[tool.cibuildwheel.environment]
+PATH = "$PATH:$HOME/.cargo/bin"
 
 [build-system]
-requires = ["poetry-core>=1.0.0", "setuptools"]
+requires = [
+    "poetry-core>=1.0.0",
+    "setuptools",
+    "maturin~=0.13",
+    "PyQt5~=5.15",
+    "Babel~=2.10",
+    "docutils~=0.17",
+    "pip"
```

Comment:
Maybe add a comment advising to update pip using `pip install -U pip` if the error `ERROR: To modify pip, please run the following command: [...]` appears

Reply:
done in 11bc3af